### DISCUSSION TITLE
It looks like the CPU module has missing documentation

### DISCUSSION THREAD

**Discussion by refaelsh** [polybar/polybar#2807] [Q&A] [answered]
It seems to be that the CPU module in the wiki has a missing documentation.
For example, I don't see there this: `bar-load-width` and many more.
I wanted a CPU load shown as bars, I google, and found this:
https://github.com/jaagr/dots/blob/master/.config/polybar/testing/cpu
Many options in that link are missing from the wiki.

1. Am I right?
2. If yes, if I open a PR to add some examples with screenshot, will that be accepted?

**Comment by patrick96** [comment, marked as answer]
Two things:

1. The `-width` property is a general property of progress bars. Those properties are documented on the [Formatting page](https://github.com/polybar/polybar/wiki/Formatting#progressbar), admittedly that documentation isn't great either, but it does list all the options. Nevertheless, it doesn't make sense to list all possible settings for each module, that would be around 100 settings per module. Better to list what type of components a module supports and document those component types elsewhere.
2. The wiki can be edited by anyone, PRs are not needed (or even possible in this case) and you don't need to ask for permission. If there are grave errors, I can always correct them.

Thanks for taking an interest :)
